src/EditorFeatures/Core/Implementation/Suggestions/SuggestedAction.cs

```diff
@@ -86,7 +90,39 @@ protected Task<ImmutableArray<CodeActionOperation>> GetPreviewOperationsAsync(Ca
                 async () => await CodeAction.GetPreviewOperationsAsync(cancellationToken).ConfigureAwait(false), cancellationToken);
         }
 
-        public virtual void Invoke(CancellationToken cancellationToken)
+        public void Invoke(CancellationToken cancellationToken)
```

Comment:
This needs to either WaitAndGetResult, or else be async so that the caller can.  If not, if the task throws an exception, it will go unobserved.